### PR TITLE
fix(button): icon text misalignment when clrLoading is used

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
Icon and text are misaligned vertically and horizontally when `clrLoading` is used.

Issue Number: CDE-1652

## What is the new behavior?
Icon and text are aligned vertically and horizontally  to design specification when `clrLoading` is used.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by dtsanevmw — APPROVED** [review]

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]

File: .storybook/stories/button/button-group.stories.ts [hunk 1]
```diff
@@ -49,8 +57,10 @@ const ButtonGroupTemplate: StoryFn = args => ({
             <clr-button
               *ngFor="let _ of createArray(buttonCount); let i = index"
               [clrInMenu]="false"
+              [clrLoading]="loading"
```
[the remark this answers is not in the document]
Comment (valentin-mladenov) [reply .storybook/stories/button/button-group.stories.ts]: Corrected in commit 839acea

File: .storybook/stories/button/button-group.stories.ts [hunk 2]
```diff
@@ -68,6 +78,14 @@ const ButtonGroupTemplate: StoryFn = args => ({
 export const ButtonGroup: StoryObj = {
   render: ButtonGroupTemplate,
 };
+export const ButtonGroupLoading: StoryObj = {
```
[the remark this answers is not in the document]
Comment (valentin-mladenov) [reply .storybook/stories/button/button-group.stories.ts]: Corrected in commit 839acea

[the remark this answers is not in the document]
Comment (valentin-mladenov) [reply .storybook/stories/button/button-group.stories.ts]: Why would that matter, anyway? 

Button group `clr-button` have totally different implementation than regular `button`.
When set to true `clrLoading` in `clr-button` just adds the spinner next to icon and text. 

The real issue is with regular buttons `clrLoading` where we have separate story.

File: .storybook/stories/button/button-group.stories.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinbuhmann) [reply .storybook/stories/button/button-group.stories.ts]: Oh, okay. It doesn't matter then. I thought it was using the loading directive. You can revert your fix if you want.

File: .storybook/stories/button/button-group.stories.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (kevinbuhmann) [reply .storybook/stories/button/button-group.stories.ts]: Thanks!

**Review by kevinbuhmann — APPROVED** [review]

**Pull request merged**